pkg/userop/userop.go

```diff
@@ -151,58 +151,56 @@ func (op UserOperation) MarshalJSON() ([]byte, error) {
 }
 
 // UnmarshalJSON decodes a JSON encoding into a UserOperation.
-func UnmarshalJSON(data []byte) (*UserOperation, error) {
```

Comment:
in my opinion, it's better to move logic into separate private function and leave both of functions (op.UnmarshalJSON and UnmarshalJSON) beacuse removing of public function in such pkg as `userop` could lead to compatibility issue in pkgs that use this particular function

Reply:
The previous `UnmarshalJSON` function was introduced in the previous PR and currently is not a dependency to any services.